web/viewer.css

```diff
@@ -920,6 +920,24 @@ html[dir="rtl"] #viewOutline.toolbarButton::before {
   content: url(images/toolbarButton-search.png);
 }
 
+.toolbarButton.pdfSidebarNotification::after {
+  position: absolute;
+  display: inline-block;
+  top: 0px;
+  /* Create a filled circle, with a diameter of 10 pixels, using only CSS: */
+  content: '';
+  background-color: #8BE651;
+  height: 10px;
```

Comment:
I think `9px` is perfect. It's very nitpicky, but I feel `8px` is too small and `10px` is slightly too large, now that they're all visualized. I don't think we should change the tooltip. The visual notification is enough to show the user that attachments/outlines exist, and people generally don't know about/don't notice tooltips (as we have seen before with questions about outline tree toggling).